src/ReactiveUI.Winforms/ActivationForViewFetcher.cs

```diff
@@ -44,12 +46,38 @@ public IObservable<bool> GetActivationForView(IActivatable view)
                 }
 
                 return controlActivation;
-            }
-            else {
+            } else {
                 // Show a friendly warning in the log that this view will never be activated
                 this.Log().Warn("Expected a view of type System.Windows.Forms.Control but it is {0}.\r\nYou need to implement your own IActivationForViewFetcher for {0}.", view.GetType());
                 return Observable<bool>.Empty;
             }
         }
+
+        private bool GetCachedIsDesignMode(Control control)
+        {
+            if (_isDesignModeCache == null) {
```

Comment:
I'm personally happy with the cached value regardless, Lazy<T> just added more code in the end over this design and we don't need the multithreaded protection in this case.